src/transaction/log_append.cpp

```diff
@@ -185,7 +185,10 @@ LOG_RESET_APPEND_LSA (const LOG_LSA *lsa)
   log_Gl.hdr.append_lsa = *lsa;
   log_Gl.prior_info.prior_lsa = *lsa;
 #if defined (SERVER_MODE)
-  log_Gl.get_log_prior_sender ().reset_unsent_lsa (*lsa);
+  if (is_active_transaction_server () || is_page_server ())
```

Comment:
This change here mandates a corresponding assert in `prior_sender::reset_unsent_lsa`:
```
  assert (is_active_transaction_server () || is_page_server ());
```

Reply:
It's been done and a unit test has been adapted.